lib/ansible/modules/windows/win_iis_webapppool.ps1

```diff
@@ -115,9 +114,23 @@ if ($pool)
     name = $pool.Name
     state = $pool.State
     attributes =  New-Object psobject @{}
+    enumerations =  New-Object psobject @{}
```

Comment:
IIRC these values don't differ across hosts/OS/versions, right? Including the name translation in the output is great, but not sure of the utility of the `enumerations` retval over just snapshotting and including in docs...